src/dft/backends/mklcpu/backward.cpp

```diff
@@ -33,174 +33,174 @@ namespace mkl {
 namespace dft {
 namespace mklcpu {
 
-void compute_backward_buffer_inplace_f(descriptor<precision::SINGLE, domain::REAL> &desc,
-                                       sycl::buffer<std::complex<float>, 1> &inout) {
-    throw std::runtime_error("Not implemented for mklcpu");
-}
-void compute_backward_buffer_inplace_c(descriptor<precision::SINGLE, domain::COMPLEX> &desc,
-                                       sycl::buffer<std::complex<float>, 1> &inout) {
-    throw std::runtime_error("Not implemented for mklcpu");
-}
-void compute_backward_buffer_inplace_d(descriptor<precision::DOUBLE, domain::REAL> &desc,
-                                       sycl::buffer<std::complex<double>, 1> &inout) {
-    throw std::runtime_error("Not implemented for mklcpu");
-}
-void compute_backward_buffer_inplace_z(descriptor<precision::DOUBLE, domain::COMPLEX> &desc,
-                                       sycl::buffer<std::complex<double>, 1> &inout) {
-    throw std::runtime_error("Not implemented for mklcpu");
-}
+// void compute_backward_buffer_inplace_f(descriptor<precision::SINGLE, domain::REAL> &desc,
+//                                        sycl::buffer<std::complex<float>, 1> &inout) {
+//     throw std::runtime_error("Not implemented for mklcpu");
```

Comment:
Shouldn't these all be `mkl::unimplemented`?

Reply:
not sure, these are your implimentations